clj-refactor.el

```diff
@@ -2228,10 +2228,14 @@ root."
   (interactive)
   (cljr--assert-leiningen-project)
   (cljr--assert-middleware)
-  (save-buffer)
-  (let ((result (nrepl-send-sync-request
+  (when (not (boundp 'filename))
+    (save-buffer))
+  (let* ((path-to-file (if (boundp 'filename)
```

Comment:
I believe this is constantly false.  I can't find a variable named `filename` when I check.